src/SharedUtilities/drivers.jl

```diff
@@ -297,27 +285,17 @@ function snow_precipitation(atmos::PrescribedAtmosphere, p, t)
     return atmos.snow_precip(t)
 end
 
-"""
-    surface_air_density(atmos::PrescribedAtmosphere,
-                        p,
-                        t,
-                        name::Symbol,
-                        earth_param_set,
-                        )
-
-Returns the air density (kg/m^3) at the surface.
-"""
+# stubs
+function surface_temperature(model::AbstractModel, Y, p) end
 function surface_air_density(
-    atmos::PrescribedAtmosphere,
+    atmos::AbstractAtmosphericDrivers,
+    model::AbstractModel,
+    Y,
     p,
     t,
-    name::Symbol,
-    earth_param_set,
-)
-    thermo_params = LSMP.thermodynamic_parameters(earth_param_set)
-    ts_in = construct_atmos_ts(atmos, t, thermo_params)
-    compute_ρ_sfc.(Ref(thermo_params), Ref(ts_in), getproperty(p, name).T_sfc)
-end
-
-
-end
+    T_sfc,
+) end
+function surface_specific_humidity(model::AbstractModel, Y, p, T_sfc, ρ_sfc) end
```

Comment:
Why do we have these stub functions? Do they get extended by individual models, or will they be implemented later?

Reply:
they do get extended, and will be available for import by the coupler when computing surface properties